gulpfile.js

```diff
@@ -1,75 +1,28 @@
-var gulp = require('gulp'),
-    rename = require('gulp-rename'),
-    sass = require('gulp-sass'),
-    autoprefixer = require('gulp-autoprefixer'),
-    notify = require('gulp-notify'),
-    gutil = require('gulp-util'),
-    scsslint = require('gulp-scss-lint'),
-    cssnano = require('gulp-cssnano');
-    sassdoc = require('sassdoc'),
-    util = require('util');
+'use strict'
 
-/* Helper functions */
-function throwSassError(sassError) {
-    throw new gutil.PluginError({
-        plugin: 'sass',
-        message: util.format(
-            "Sass error: '%s' on line %s of %s",
-            sassError.message,
-            sassError.line,
-            sassError.file
-        )
-    });
-}
+require('es6-promise').polyfill();
 
-/* Gulp instructions start here */
-gulp.task('help', function() {
-    console.log('sass - Generate the min and unminified css from sass');
-    console.log('docs - Generate the docs from the source sass');
-    console.log('build - Generate css');
-    console.log('watch - Watch sass files and generate unminified css');
-    console.log('test - Lints Sass');
-});
-
-gulp.task('sasslint', function() {
-    var path = (gutil.env.file)? gutil.env.file : '**/*.scss';
-    return gulp.src('scss/' + path)
-        .pipe(scsslint({ 'config': '.scss-lint.yml' }))
-        .pipe(scsslint.failReporter());
-});
-
-gulp.task('sass', function() {
-    return gulp.src('scss/**/*.scss')
-        .pipe(sass({
-            style: 'expanded',
-            errLogToConsole: true,
-            onError: throwSassError
-        }))
-        .pipe(autoprefixer('last 2 version', 'safari 5', 'ie 8', 'ie 9', 'opera 12.1'))
-        .pipe(gulp.dest('build/css/'))
-        .pipe(rename({suffix: '.min'}))
-        .pipe(cssnano())
-        .pipe(gulp.dest('build/css/'));
-});
+var gulp = require('gulp');
+var wrench = require('wrench');
 
-gulp.task('docs', function() {
-    return gulp.src('scss/**/*.scss')
-        .pipe(sassdoc({ 'dest': 'build/docs'}));
+/*
+  concatenate all *.js / *.coffee files in the 'gulp' folder
+ */
+wrench.readdirSyncRecursive('./gulp').filter(function(file) {
```

Comment:
I can see this theme having JS in the future. Especially if there are react / angular components from the respective products. For the time being I will remove this.